serving/docker/scripts/install_tensorrt.sh

```diff
@@ -1,47 +0,0 @@
-#!/bin/bash
```

Comment:
i am leveraging this script in the internal build, should we remove that too?

Reply:
no, internal build you still need trt

Reply:
i meant if we are okay to use it from legal point of view

Reply:
to build it is ok, but we have some issues with libnvinfer.so